### PR TITLE
Autotests: #3053 Autotests limited automation cases for atom tool

### PR DESCRIPTION
**Pull request by AleksandraSmolianinova**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"

### CODE REVIEW

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Atom/Atom-Tool/group-generics-buttons-check.spec.ts [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    await clickOnAtom(page, 'ACY', 0);
+  });
+
+  test('Acylic Carbo atoms adding to the atom of structure', async ({
+    page,
+  }) => {
+    /*
+        Test case: EPMLSOPKET-1508
+      */
+    await selectAtomInToolbar(AtomButton.Extended, page);
```
Comment (Nitvex) [inline ketcher-autotests/tests/Structure-Creating-&-Editing/Atom/Atom-Tool/group-generics-buttons-check.spec.ts:70]: I propose to create a separate test/functions for each atom

**Review by Nitvex — CHANGES_REQUESTED** [review]

**Comment by y-holik** [comment]
1 test failing - 
image

retested locally (passed)
image

**Review by Zhirnoff — APPROVED** [review]

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Atom/Atom-Tool/group-generics-buttons-check.spec.ts [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    await page.getByRole('button', { name: 'ABC', exact: true }).click();
+    await page.getByTestId('OK').click();
+    await clickOnAtom(page, 'S', 0);
+
+    await selectAtomInToolbar(AtomButton.Extended, page);
```
Comment (Nitvex) [inline ketcher-autotests/tests/Structure-Creating-&-Editing/Atom/Atom-Tool/group-generics-buttons-check.spec.ts:72]: Can we create a separate function for these repetitive steps?

Comment (y-holik) [reply ketcher-autotests/tests/Structure-Creating-&-Editing/Atom/Atom-Tool/group-generics-buttons-check.spec.ts:72]: done

**Review by Nitvex — APPROVED** [review]

**Pull request merged**